src/PlatformSockets.c

```diff
@@ -10,24 +6,18 @@
 
 void addrToUrlSafeString(struct sockaddr_storage* addr, char* string)
 {
-    char addrstr[48];
-#ifdef __vita__
-    struct sockaddr_in* sin = (struct sockaddr_in*)addr;
-    sceNetInetNtop(addr->ss_family, &sin->sin_addr, addrstr, sizeof(addrstr));
-    if (addr->ss_family == AF_INET6) {
-        sprintf(string, "[%s]", addrstr);
-    } else {
-        sprintf(string, "%s", addrstr);
-    }
-#else
     char addrstr[INET6_ADDRSTRLEN];
 
     if (addr->ss_family == AF_INET6) {
+#ifdef __vita__
+        LC_ASSERT(FALSE);
```

Comment:
Unless you have strong objections, I'd prefer to remove the #ifdef __vita__ and Vita-only LC_ASSERTs from the IPv6 related code paths in this file. As long as you're using an IPv4-only DNS lookup routine like gethostbyname(), you'll never get IPv6 addresses anyway. I'd like to minimize the Vita-specific code (since I won't be able to easily test it, among other reasons).